package.json

```diff
@@ -103,6 +103,7 @@
 		"lint:css": "stylelint \"**/*.{css,scss}\"",
 		"lint:js": "eslint --ext .js,.jsx,.ts,.tsx,.mjs,.json --cache .",
 		"lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"!build/**\" \"!node_modules/**\" \"!public/**\"",
+		"lint:unused-state-action-types": "node bin/find-unused-state-action-types.sh",
```

Comment:
This way, this script will only run in CI, right? Any easy way to make it run locally and display linting errors in the IDE?

Reply:
> This way, this script will only run in CI, right? Any easy way to make it run locally and display linting errors in the IDE?

Yeah, as currently implemented, I'm expecting this will most often be flagged through CI, though technically you can run the lint script locally as well.

An editor integration would be nice to have. I'd imagine that'd be implemented as a custom ESLint rule? The proposed direction felt like a good middle-ground of enforcing an expectation without going too deep into a rabbit hole about how to surface it. It's likely to be a pretty uncommon scenario, and addressing a CI failure wouldn't be overly burdensome IMO. I'm also conscious of performance here since even the low-level `grep`'ing was surprisingly slow without parallelization.